--- libexif-gtk-0.3.5-orig/libexif-gtk/Makefile.am	2004-10-17 08:48:35.000000000 -0500
+++ libexif-gtk-0.3.5/libexif-gtk/Makefile.am	2006-01-11 13:59:52.812500000 -0600
@@ -41,7 +41,7 @@
 	gtk-exif-entry-user-comment.c		\
 	gtk-exif-entry-version.c		\
 	gtk-exif-util.h
-libexif_gtk_la_LDFLAGS = -version-info @LIBEXIF_GTK_VERSION_INFO@
+libexif_gtk_la_LDFLAGS = -version-info @LIBEXIF_GTK_VERSION_INFO@ -no-undefined
 libexif_gtk_la_LIBADD =						\
 	$(top_builddir)/gtk-extensions/libgtk-extensions.la	\
 	$(LIBEXIF_GTK_LIBS)
